### PR TITLE
enable validation on Learn included markdown

### PR DESCRIPTION
**Pull request by fkpwolf**
[AB#300879](https://dev.azure.com/ceapex/d3d54af3-265a-4f18-95f6-9a46397ca583/_workitems/edit/300879)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/docfx/pull/6561)

### CODE REVIEW

**Comment by fkpwolf** [comment]
@yufeih @OsmondJiang the diff is excepted since we enabled validation on included marddown for Learn.

File: src/docfx/validation/ContentValidator.cs [hunk 1]
```diff
@@ -15,7 +15,8 @@ namespace Microsoft.Docs.Build
     internal class ContentValidator
     {
         // Now Docs.Validation only support conceptual page, redirection page and toc file. Other type will be supported later.
-        private static readonly string[] s_supportedPageTypes = { "conceptual", "includes", "toc", "redirection" };
+        // Learn content: "learningpath", "module", "moduleunit"
+        private static readonly string[] s_supportedPageTypes = { "conceptual", "includes", "toc", "redirection", "learningpath", "module", "moduleunit" };
```
Comment (yufeih) [inline src/docfx/validation/ContentValidator.cs:19]: Update the page type mapping to include "learn", base on `docs-ui`, these are the `learn` schemas:
image


```suggestion
        private static readonly string[] s_supportedPageTypes = { "conceptual", "includes", "toc", "redirection", "learn" };
```

Comment (fkpwolf) [reply src/docfx/validation/ContentValidator.cs:19]: If we use `Learn` as page type, then in below dialog we will have to merge all `Learn.*` entries into only one entry `Learn`? @OsmondJiang 
image

Comment (OsmondJiang) [reply src/docfx/validation/ContentValidator.cs:19]: Megan require to use Page kind instead of Page type

**Review by yufeih — APPROVED** [review]

**Pull request merged**